financial-services/1.10/modules/ROOT/pages/rpa-setup-guide.adoc

```diff
@@ -0,0 +1,145 @@
+. Save and upload the project to RPA Manager.
+
+== Transferring Project to the Test Phase and Testing It
+
+Now that you have adapted the file to your AMS, xref:rpa-manager::processautomation-deploy.adoc#test-a-process-automation[follow these steps] to transfer the project to the Test phase and test it.
+
+== Transferring the Project to the Production Phase
+
+After successfully testing the project, xref:rpa-manager::processautomation-deploy.adoc[transfer the project to the Production phase]. Once transferred, do one of the following:
+
+* xref:rpa-manager::processautomation-deploy.adoc#deploy-a-process-automation[Deploy the project] and schedule it as needed.
+* Publish the project to Anypoint Exchange as an invocable run configuration, which can be called as an invocable action in a flow.
+** xref:rpa-manager::processautomation-deploy#publish-automation-exchange[Publish RPA Manager Project to Exchange]
+** xref:rpa-manager::settings-connect.adoc[RPA Manager Settings]
+** https://help.salesforce.com/s/articleView?id=sf.flow_build_use_flows_with_mulesoft_rpa.htm&type=5[Use Flows with MuleSoft RPA]
+** xref:composer::ms_composer_rpa_reference.adoc[RPA Reference]
+)
+
+== Example of Adapting the CRPA File
+
+[DISCLAIMER]
+This example uses real world data to show you how to adapt a .crpa file to a specific AMS.
+
+All values for the Activity Parameters used in RPA Builder are test values that you must overwrite with your own values when creating the run configuration for the process in RPA Manager.
+
+=== Process Model
+
+The process model contains the following three activities:
+
+. Log in to the AMS using credentials and a database name.
+. Open a record in the AMS database using a lookup code.
+. Copy data from various fields on the record pages to [should be a CSV file but is a POST REST call in JSON format to http://mainstreet-demo.us-e2.cloudhub.io/epicpolicy]
+
+image placeholder
+
+You can add activities by clicking on an element on the model panel and choosing an element from the context menu. For example, the following image shows how to branch off to a second AMS if you are using more than one and not finding the data you need in the first AMS:
+
+image placeholder
+
+Click on an activity to open the editing workflow.
+
+=== Activity 1: Log in to the AMS
+
+Follow these steps to log in to the AMS:
+
+. Decrypt the AMS login credentials that go in through an Activity Parameter.
+. Run the AMS.
+. Log in to the AMS using the decrypted credentials.
+. Select the database using another in-going Activity Parameter.
++
+image placeholder
+
+. Change the following in the *Run Program Wizard* dialog:
+
+.. Replace the values in the *File name* and *Window Title* fields with the values from your AMS.
+.. Click the *Test* button to run the application so that you can use it to adapt the following Action Steps:
++
+image placeholder 1
++
+image placeholder 2
```

Comment:
Hi @tasmoore, you only need one of each duplicated image. If you like them, you can use the marked-up screenshots. If not, use the blank ones and add your own mark-up.